docs/serving/autoscaling.md

```diff
@@ -179,11 +180,39 @@ template.
 spec:
   template:
     metadata:
-      autoscaling.knative.dev/metric: concurrency
+      autoscaling.knative.dev/metric: cpu
       autoscaling.knative.dev/class: hpa.autoscaling.knative.dev
 ```
 
-## Additional resources
+# Implementing your own Pod Autoscaler
+
+The Pod Autoscaler custom resource allows you to implement your own autoscaler without changing anything else about the Knative Serving system.
+
+You can implement your own Pod Autoscaler if the requirements of your workload cannot be covered by the KPA or HPA, for example if you want to use a more specialized autoscaling algorithm or if you need to use a specialized set of metrics not supported by Knative out of the box.
```

Comment:
What are the other "requirements" for workloads that cannot be covered by the KPA or HPA? Or, are there really just the two scenarios that this feature is targeting?

Reply:
@markusthoemmes care to provide any insight here? I think it's just the two scenarios listed?

@RichieEscarez I might be missing something but I don't see anything wrong with how this is stated?

Reply:
can't think of more off the top of my head. Plus it's qualified as "for example" so that should leave some leeway?